docs/Troubleshooting.md

```diff
@@ -27,8 +27,9 @@ rollup: frontend/main.js
 
 We're not sure why this happens, but there are a few solutions:
 
-- Create a newline near the top of the file with a comment and save again
-- Restart `grunt dev`, or do `grunt clean dev` to also delete the sourcemaps
+- A. Change `.Gruntfile.babel.js` line 259 `sourcemap` key to `false` and restart `grunt dev`.
```

Comment:
Make sure to add here that it needs to be switched back before any pull requests are submitted

Reply:
A said it on the file itself. But I can enforce it here too. Done.